controllers/cloud.redhat.com/providers/dependencies/impl.go

```diff
@@ -130,6 +137,7 @@ func processAppEndpoints(
 					Name:     innerDeployment.Name,
 					App:      depApp.Name,
 					TlsPort:  utils.IntPtr(int(tlsPort)),
+					ApiPath:  fmt.Sprintf("/api/%s/", apiPath),
```

Comment:
Not sure I like the root of the path being hardcoded like this ( I mean, forcing the endpoint to always be `/api/your-path` ) 

I understand it's heavily unlikely that this changes on Stage and Prod - however unlikely, I think this part (the ROOT api) should be something configurable. I anything, could this be set at a "higher" level (ClowdEnvironment level) and picked up from there ? I don't know if this makes sense, ultimately I'd like to avoid having to change the source code if we change the root path.

Reply:
I think we can look at this further down the line - I'm not afraid of putting a configuration item in later. But making it a mandatory field right now is a no-go and then we need yet more discussions on how to implement this. Unfortunately `/api` is already hardcoded in other places. Yes it's non-ideal but I think if we fix it, we fix it everywhere of course, and I don't want to do that in this PR.